configure.ac

```diff
@@ -146,6 +148,9 @@ PKG_CHECK_MODULES([LEVELDB], [leveldb], ,
         ],
         AC_MSG_ERROR(['leveldb is required to build unifycr.'])))
 
+# look for gotcha library, sets GOTCHA_INCLUDE, GOTCHA_LIB
+X_AC_GOTCHA
```

Comment:
It might be helpful to use a more descriptive prefix like UNIFYCR_ for our m4 macros to make internal ones easy to identify.